.travis.yml

```diff
@@ -0,0 +1,120 @@
+# Use the R language default, as travis always has Python
+language: python
```

Comment:
I initially tried R.  Fundamentally, we use R, python, javascript, and bash.  If you use the travis R environment, the python is harder to get to what we want then it was to specify python and get R where we want.  Python is a first-class citizen in travis, whereas R is still a second-class citizen.